docs/04-FunctionalTests.md

```diff
@@ -250,10 +267,43 @@ error_level: "E_ALL & ~E_STRICT & ~E_DEPRECATED"
 
 `error_level` can also be set globally in `codeception.yml` file.
 
+## Accessing external URL's with Guzzle
```

Comment:
The code is right and the suggestion works. However, it should not be the part of this specific chapter. This is a general advice not actually related to functional tests. It should be placed into [Cookbook](http://phptest.club/c/codeception/cookbook)